### PR TITLE
Add sentry

### PR DESCRIPTION
**Pull request by aexvir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,13 @@ Like above, but no docker.
 
 Like above, but s4cmd instead of s3cmd
 
+## kiwicom/sentry
+
+- Base image: [`sentry:9.0.0`](https://hub.docker.com/_/sentry/)
```
Comment (underyx) [inline README.md]: We'll forget to update this for every patch version, maybe just mention `sentry` alone?

File: sentry/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM sentry:9.0.0
+
+LABEL name=sentry version=$SENTRY_VERSION \
```
Comment (underyx) [inline sentry/Dockerfile]: `$SENTRY_VERSION` is set on the instruction right below

Comment (aexvir) [reply sentry/Dockerfile]: I wanted to :sonic: fix it but failed :(

File: README.md (hunk 1 above)
Comment (aexvir) [reply README.md]: 🆗

**Pull request merged**